helm-chart/templates/service.yaml

```diff
@@ -8,9 +8,12 @@ metadata:
   annotations:
 {{ toYaml .Values.service.annotations | nindent 4 }}
 {{- end }}
-{{- if .Values.service.annontations_JSON }}
+# This addes support for an NEG setup where the key is a string but the value is JSON
+{{- if .Values.service.annotations_NEG }}
   annotations:
-{{ .Values.service.annontations_JSON | toJson | nindent 4 }}
+{{- range  $key, $value := .Values.service.annotations_NEG }}
```

Comment:
Why is a separate `annotations_NEG`/`annotations_JSON` necessary? There's no need to escape the JSON, just use double quotes wrapped in single quotes like the mirror node does in [production](https://github.com/hashgraph/hedera-mirror-node/blob/1625a0414c28f0e3b82b0bb3f4765f659000439d/clusters/mainnet-na/common/helmrelease.yaml#L58) to populates its NEG.